backend/remote-state/http/client_test.go

```diff
@@ -26,7 +26,7 @@ func TestHTTPClient(t *testing.T) {
 
 	url, err := url.Parse(ts.URL)
 	if err != nil {
-		t.Fatalf("err: %s", err)
+		t.Fatalf("Parse: %s", err)
```

Comment:
Why this change from err to Parse?

Reply:
The change from `err` to `Parse` is because `err` doesn't do much to clarify the situation. We already know it's an error, so nothing is being gained by adding `err`. In the codebase of Go itself, usually what is done is to prefix the error with the name of the function that generated the error, so that it's clearer at-a-glance what broke. For an example, see: https://go-review.googlesource.com/c/go/+/205247/4/src/net/smtp/smtp_test.go

Reply:
Neat, I like it.